Configs/.config/hypr/scripts/batterynotify.sh

```diff
@@ -19,8 +19,8 @@ while (( "$#" )); do  # Parse command-line arguments and defaults
       echo "  --low, -l         Set battery low threshold (default: $mnl% percent)"
       echo "  --unplug, -u      Set unplug charger threshold (default: $mxu% percent )"
       echo "  --timer, -t       Set countdown timer (default: $mnt seconds)"
-      echo "  --interval, -i    Set notify interval  on LOW UNPLUG Status  (default: $mni% percent)"
-      echo "  --notify, -n      Set notify interval for Battery Full Status  (default: $mnn minutes)"
+      echo "  --interval, -i    Set notify interval  on LOW UNPLUG Status  (default: 2% percent)"
```

Comment:
2% is a very critical battery level. especially on old laptops, they'd just power off. Some battery would start to count down in seconds alongside their current capacity eg. starting from 10% and would black screen before they even reached that low capacity.
I'd suggest the Low battery would start at 20% with <x time> interval notification. & critical would maybe at maybe 5% or 10% so user would have time to plug it in.